### ISSUE TITLE
Address color contrast a11y failure in page component

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-next#1691]
```
1 error found in: /components/Page/examples-full/

1) Elements must have sufficient color contrast -- https://dequeuniversity.com/rules/axe/3.1/color-contrast -- impact: serious
      
-------html violation instances-------

<div class="pf-c-page__header-tools">
            header-tools
    </div>
<div class="pf-c-page__header-nav">
            pf-c-nav
    </div>
<div class="pf-c-page__header-tools">
            header-tools
    </div> 
```
